### PR TITLE
Xpanse packs for cortex

### PR DESCRIPTION
**Pull request by mitesh-pant**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yaakovpraisler** [comment]
@anara123 - I'll review this PR once [this ticket](https://jira-hq.paloaltonetworks.local/browse/CIAC-3653) is done.

File: Tests/Marketplace/marketplace_constants.py [hunk 1]
```diff
@@ -202,6 +211,8 @@ class PackFolders(enum.Enum):
     CORRELATION_RULES = 'CorrelationRules'
     XSIAM_DASHBOARDS = 'XSIAMDashboards'
     XSIAM_REPORTS = 'XSIAMReports'
+    XPANSE_DASHBOARDS = 'XPANSEDashboards'
+    XPANSE_REPORTS = 'XPANSEReports'
```
Comment (daryakoval) [inline Tests/Marketplace/marketplace_constants.py:215]: I think it is not relevant.

Comment (mitesh-pant) [reply Tests/Marketplace/marketplace_constants.py]: updated

**Comment by yaakovpraisler** [comment]
This is done as part of [this PR](https://github.com/demisto/content/pull/22090).
Closing.

**Pull request closed without merging**